### PR TITLE
EPMLABSBRN-899 change audiometry dto, fix loading sin audiometry

### PR DESCRIPTION
**Pull request by ElenaSpb**
[EPMLABSBRN-899](https://jira.epam.com/jira/browse/EPMLABSBRN-899)

change dto, fix loading sin audiometry

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt [hunk 1]
```diff
@@ -92,6 +92,11 @@ class AudiometryLoader(
     fun loadFrequencyDiagnosticData() {
         val audiometrics = audiometryRepository.findByAudiometryType(AudiometryType.SIGNALS.name)
         audiometrics.forEach { audiometry ->
+            val existTasks = audiometryTaskRepository.findByAudiometry(audiometry)
```
Comment (Dmitriy1Fokin) [inline src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt:95]: you can make a query in the database for the count of tasks

File: src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt [hunk 2]
```diff
@@ -92,6 +92,11 @@ class AudiometryLoader(
...
+            if (existTasks.size == 2)
+                return@forEach
+            if (existTasks.size > 2)
```
Comment (Dmitriy1Fokin) [inline src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt:98]: case when 1 existTasks is not necessary to check?

File: src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt (hunk 1 above)
Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt:95]: done

File: src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt (hunk 2 above)
Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/load/AudiometryLoader.kt]: ok, added, it is inconsistent db status too

**Review by Dmitriy1Fokin — APPROVED** [review]

**Pull request merged**